include/boost/astronomy/io/image.hpp

```diff
@@ -40,9 +41,9 @@ template <typename PixelType>
 struct image_buffer
 {
 protected:
-    std::valarray<PixelType> data; //! stores the image
-    std::size_t width; //! width of image
-    std::size_t height; //! height of image
+    std::valarray<PixelType> data_; //! stores the image
```

Comment:
This entire class will be now deprecated and replaced with boost Gil . There is a very high performance hit here